ionic.config.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "ion-ams-report",
   "app_id": "0451e327",
-  "type": "ionic-angular"
+  "type": "ionic-angular",
+  "integrations": {
```

Comment:
What's the purpose of this field (integrations) ?

Reply:
integrations was generated automatically by ionic serve cmd